### ISSUE TITLE
Failure downloading files that are visible in the web interface

### ISSUE THREAD

**Issue by jbrinchmann** [spacetelescope/crds#788]
Hi,
I am trying to get reference files for updating headers on some WFPC2 data (refid u2dk0202t). When running crds (installed with Astroconda) using

crds bestrefs --files u2d*_c0m.fits --sync-references=1 --update-bestrefs

(with CRDS_SERVER_URL=https://hst-crds.stsci.edu & CRDS_PATH=reference_files/)

I run into a problem with some files (specifically dbu1405iu.r1d, e371355iu.r5d , f151225pu.r3d, f1j1615tu.r2d, f8213081u.r0d, g6h0944pu.r4d):

`CRDS - INFO -  Fetching  reference_files/references/hst/wfpc2/g6h0944pu.r4d        10.2 M bytes  (6 / 6 files) (0 / 46.2 M bytes)`

`CRDS - ERROR -  Failure downloading file 'g6h0944pu.r4d' : Error fetching data for 'g6h0944pu.r4d' at CRDS server 'https://hst-crds.stsci.edu' with mode 'http' : Uknown file type for: 'g6h0944pu.r4d'
`

I do not know why it refers to .r?d files given that the header specifies .r?h file (e.g. FLATFILE= 'uref$g6h0944pu.r4h'), and the flatfile is visible in the web interface (e.g. https://hst-crds.stsci.edu/browse/g6h0944pu.r4h), and I also do not understand why these files fail to download - is this a problem with my installation of crds?  (given that all other files download correctly I am a bit puzzled)

**Comment by eslavich** [comment]
Hi, I'm not very familiar with GEIS files but my understanding is that each logical file consists of two actual files, one header file and one data file.  The `.r?h` file is the "header" file and the corresponding `.r?d` is the "data" file.  The dataset only refers to the GEIS header, but both are necessary which is why you're seeing download attempts for `g6h0944pu.r4d`.

As to why the download fails, I think that may be due to the bug that https://github.com/spacetelescope/crds/pull/749 proposes to fix.  I opened that PR back in June (!) and have since forgotten all about it.  I'll see about getting it merged tomorrow, but in the meantime if you'd like to try running bestrefs with my branch, you can install it like this:

```
pip install git+https://github.com/eslavich/crds.git@eslavich-recognize-geis-data-files
```

**Comment by eslavich** [comment]
I tested this today and was able to download the `.r?d` files with the changes in the branch.

**Comment by eslavich** [comment]
The fix has been released to pypi in version 10.3.5.
